client/src/reducers/session/reducer.js

```diff
@@ -13,6 +13,7 @@ const initialState = {
   forgotPasswordRequested: false,
   user: null,
   organisationId: null,
+  isNewOrganisation: false,
```

Comment:
Do we still need this?

Reply:
I forgot this one and the other. of course not.